proto/canine_chain/filetree/tx.proto

```diff
@@ -23,109 +22,100 @@ service Msg {
 message MsgPostFile {
   string creator = 1;
   string account = 2;
-  string hashParent = 3;
-  string hashChild = 4;
+  string hash_parent = 3;
+  string hash_child = 4;
   string contents = 5;
   string viewers = 6;
   string editors = 7;
-  string trackingNumber = 8;
+  string tracking_number = 8;
 }
 
 message MsgPostFileResponse { string path = 1; }
 
 message MsgAddViewers {
   string creator = 1;
-  string viewerIds = 2;
-  string viewerKeys = 3;
+  string viewer_ids = 2;
+  string viewer_keys = 3;
   string address = 4;
-  string fileowner = 5;
+  string file_owner = 5;
 }
 
 message MsgAddViewersResponse {}
 
-message MsgPostkey {
+message MsgPostKey {
   string creator = 1;
   string key = 2;
 }
 
-message MsgPostkeyResponse {}
+message MsgPostKeyResponse {}
 
 message MsgDeleteFile {
   string creator = 1;
-  string hashPath = 2;
+  string hash_path = 2;
   string account = 3;
 }
 
 message MsgDeleteFileResponse {}
 
 message MsgRemoveViewers {
   string creator = 1;
-  string viewerIds = 2;
+  string viewer_ids = 2;
   string address = 3;
-  string fileowner = 4;
+  string file_owner = 4;
 }
 
 message MsgRemoveViewersResponse {}
 
-message MsgMakeRoot {
-  string creator = 1;
-  string account = 2;
-  string rootHashPath = 3;
-  string contents = 4;
-  string editors = 5;
-  string viewers = 6;
-  string trackingNumber = 7;
-}
 
 message MsgMakeRootResponse {}
 
 message MsgMakeRootV2 {
```

Comment:
remove MsgMakeRoot, MsgMakeRootV2, and MsgMakeRootResponse. replace with discussed MsgProvisionFileTree and MsgProvisionFileTreeResponse

Reply:
resolved